include/matx/transforms/convert/sparse2dense_cusparse.h

```diff
@@ -215,33 +215,20 @@ using sparse2dense_cache_t =
 
 } // end namespace detail
 
-template <typename Op>
```

Comment:
What was the reason for removing this? Does cusparse support every type of input? This would include all types, strides, etc

Reply:
checking on type is still a TODO (waiting for my tests);
it was auto removed, but I put it back for later use